src/store/modules/accounts/index.js

```diff
@@ -14,65 +8,16 @@ export default {
   namespaced: true,
   modules,
 
-  state: {
-    list: [{
-      globalIdx: 0,
-      idx: 0,
-      showed: true,
-      type: ACCOUNT_HD_WALLET,
-      protocol: PROTOCOL_AETERNITY,
-    }],
-    activeIdx: 0,
-  },
-
-  getters: {
-    active: ({ list, activeIdx }) => list[activeIdx],
-    getByType: ({ list }) => (type) => list.filter((acc) => acc.type === type),
-    getModule: () => ({ type }) => {
-      const [name, module] = Object.entries(modules)
-        .find(([, { account }]) => account.type === type)
-        || (() => { throw new Error(`Unknown account type: ${type}`); })();
-      return { ...module, name };
-    },
-  },
-
-  mutations: {
-    setActiveIdx(state, activeIdx) {
-      state.activeIdx = activeIdx;
-    },
-    add(state, account) {
-      state.list.push({
-        showed: state.list.reduce((a, b) => (b.showed ? a + 1 : a), 0) < 8,
-        ...account,
-      });
-      if (!account.isRestored) {
-        state.activeIdx = state.list.length - 1;
-      }
-    },
-    remove(state, idx) {
-      if (state.activeIdx === state.list.length) {
-        state.activeIdx = 0;
-      }
-      delete state.list[idx];
-    },
-    toggleShowed(state, idx) {
-      if (state.activeIdx === idx) {
-        state.activeIdx = 0;
-      }
-      state.list[idx].showed = !state.list[idx].showed;
-    },
-  },
-
   actions: {
-    sign({ getters: { active, getModule }, dispatch }, { data, options }) {
-      return dispatch(`${getModule(active).name}/sign`, { data, options });
+    sign({ dispatch }, { data, options }) {
+      return dispatch('hdWallet/sign', { data, options });
     },
 
-    signTransaction({ getters: { active, getModule }, dispatch }, { txBase64, options }) {
+    signTransaction({ dispatch }, { txBase64, options }) {
       if (options && options.fromAccount) {
-        return dispatch(`${getModule(active).name}/signTransactionFromAccount`, { txBase64, options });
+        return dispatch('hdWallet/signTransactionFromAccount', { txBase64, options });
       }
-      return dispatch(`${getModule(active).name}/signTransaction`, { txBase64, options });
+      return dispatch('hdWallet/signTransaction', { txBase64, options });
```

Comment:
We may support only `hdWallet` for now but we had the provisions in place to support ledger wallet. I am not sure why this PR is dropping the support of hardware ledger accounts.

Reply:
We didn't support the ledger. Please take a closer look on the PR and this line: https://github.com/aeternity/superhero-wallet/blob/develop/src/store/modules/accounts/index.js#L11

Reply:
Sure. It's better to have a module-based routing but that can be introduced when adding ledger wallet feature too.